python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -118,11 +118,23 @@ def add_n(attrs, inputs, proto_obj):
 # Sorting and Searching
 def argmax(attrs, inputs, proto_obj):
     """Returns indices of the maximum values along an axis"""
-    return 'argmax', attrs, inputs
+    new_attrs = translation_utils._fix_attribute_names(attrs, {'axis': 'axis',
+                                                               'keepdims': 'keepdims'})
+    axis = new_attrs.get('axis', 0)
+    keepdims = new_attrs.get('keepdims', 0)
+    argmax_op = symbol.argmax(inputs[0], axis=axis, keepdims=keepdims)
+    cast_attrs = {'dtype': 'int64'}
```

Comment:
why are we casting the result to int64?

Reply:
argmax/argmin output is always int64 in onnx https://github.com/onnx/onnx/blob/master/docs/Operators.md#ArgMax
Tests will fail otherwise

Reply:
Please add it as code comment for easy maintenance.